src/main/java/com/teamacronymcoders/base/subblocksystem/blocks/BlockSubBlockHolder.java

```diff
@@ -164,4 +167,75 @@ public int damageDropped(@Nonnull IBlockState state) {
     public List<IGeneratedModel> getGeneratedModels() {
         return this.getSubBlocks().values().stream().map(ISubBlock::getGeneratedModel).filter(Objects::nonNull).collect(Collectors.toList());
     }
+
+    @Override
+    public boolean isSideSolid(IBlockState blockState, @Nonnull IBlockAccess world, @Nonnull BlockPos pos, EnumFacing side) {
+        return this.getSubBlock(blockState).isSideSolid();
```

Comment:
You should probably pass the side in this one, just in case in the future for slabs or something.